pkg/api/kubegraph/nodes/types.go

```diff
@@ -228,13 +228,18 @@ type SecretNode struct {
 	osgraph.Node
 	*kapi.Secret
 
-	IsFound bool
+	IsFound     bool
+	IsForbidden bool
 }
 
 func (n SecretNode) Found() bool {
 	return n.IsFound
 }
 
+func (n SecretNode) Forbidden() bool {
```

Comment:
Seems like this is likely to be generic, but the question is, is it generic to the node or generic to resource kind as a context when creating edges.  Seems more likely to be generic as context on edge creation.